apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Plugin Name: Full Site Editing
+ */
+
+class WPCOM_Full_Site_Editing {
+	static $initialized = false;
+
+	function __construct() {
+		if ( self::$initialized ) {
+			return;
+		}
+		self::$initialized = true;
+
+		$this->register_script_and_style();
+
+		add_action( 'init', array( $this, 'register_blocks' ), 100 );
+	}
+
+	function register_script_and_style() {
+		wp_register_script(
+			'wpcom-full-site-editing-script',
+			plugins_url( 'dist/full-site-editing-plugin.js', __FILE__ ),
+			array(
```

Comment:
You could add `wp-polyfill` dependency here since the build doesn't take care of that for you. Pretty much every other of your dependencies already relies on it implicitly but it's always nice to be explicit. :-)